### PR TITLE
add tests and configure prettier

### PR DESCRIPTION
**Pull request by BolajiOlajide**

### CODE REVIEW

File: src/streamers/spotify.streamer.ts [hunk 1]
```diff
@@ -1,146 +1,200 @@
...
-      this.storer.set(SPOTIFY_TRACK_KEY, lastPlayed, this.cacheDuration);
+      this.storer.set(SPOTIFY_TRACK_KEY, lastPlayed, this.cacheDuration)
     }
 
-    return lastPlayed;
+    return lastPlayed
+  }
+
+  setUseCache(useCache: boolean): void {
+    this.useCache = useCache
```
Comment (lilpolymath) [inline src/streamers/spotify.streamer.ts:198]: This function is not in use.

File: src/streamers/__tests__/spotify.streamer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+      expect(cachedSong).toBeUndefined()
+      expect(response).toEqual({
+        is_playing: false,
+        title: 'Coming Home',
+        artiste: 'Kanye West',
+        image_url: 'https://i.scdn.co/image/ab67616d0000b273712549143',
+        preview_url:
+          'https://p.scdn.co/mp3-preview/0663627c27885467868491a91185643b0508975c?cid=774b29d4f13844c495f206cafdad9c86',
+        url: 'https://open.spotify.com/track/6M2wZ9GZgrQXHCFfjv46we',
+      })
```
Comment (lilpolymath) [inline src/streamers/__tests__/spotify.streamer.spec.ts]: I think we can move this and others to a data file, it is repeated in quite a number of places.

**Review by lilpolymath — CHANGES_REQUESTED** [review]

File: src/streamers/__tests__/spotify.streamer.spec.ts (hunk 2 above)
Comment (BolajiOlajide) [reply src/streamers/__tests__/spotify.streamer.spec.ts]: Yeah, I'll create a fixture file for this.

File: src/streamers/spotify.streamer.ts (hunk 1 above)
Comment (BolajiOlajide) [reply src/streamers/spotify.streamer.ts:198]: I use it in the test to change the value of `useCache`. Check https://github.com/BolajiOlajide/now-playing/blob/bo/add-tests-configure-prettier/src/streamers/__tests__/spotify.streamer.spec.ts#L276

File: src/streamers/__tests__/spotify.streamer.spec.ts (hunk 2 above)
Comment (BolajiOlajide) [reply src/streamers/__tests__/spotify.streamer.spec.ts]: Done

**Review by lilpolymath — APPROVED** [review]

**Pull request merged**